libs/galley-types/src/Galley/Types/Teams/Intra.hs

```diff
@@ -59,3 +60,13 @@ instance FromJSON TeamStatusUpdate where
 
 instance ToJSON TeamStatusUpdate where
     toJSON s = object ["status" .= tuStatus s]
+
+newtype TeamName = TeamName
+    { tnName :: Text }
+
+instance FromJSON TeamName where
```

Comment:
we could derive Generic and get FromJSON "for free"

Reply:
👍